Makefile

```diff
@@ -214,7 +220,7 @@ endif
 clean:
 	$(RM) -r $(BUILD_DIR)
 
-distclean:
+distclean: distclean_kart_assets
```

Comment:
I'm going to test distclean before I merge.

Reply:
Sure thing. All it SHOULD do is clean out the png/bin/mio0 file from the assets folder. Regular `clean` cleans out the build directory. I forget what all `distclean` does but it includes deleting the textures the original `extract_assets` exported. So `distclean_assets` is intended to be a super target that allows you to clean out the assets extracted by the new utiltity.

In the future I plan to add stuff like `distclean_lakitu`, `distclean_karts`, etc to the appropriate `.mk` file so you could target a specific group of assets. For now though the "burn it all" option is the only one available

Reply:
Should the distclean_kart_assets part be removed? Or did I do something wrong?
```
make distclean
make: *** No rule to make target 'distclean_kart_assets', needed by 'distclean'.  Stop.
```
Perhaps you meant to put `$(MAKE) distclean_assets`?

For distclean_assets. Can we add make clean?
```
distclean_assets:
	rm -rf $(ASSET_DIRECTORIES)
	$(MAKE) clean
```
This will help the user-experience. If you build after removing the assets it will error due to not realizing it needs to rebuild everything which could be confusing.

Reply:
Also,

Please comment out the call to `$(DLSYMGEN)` and `$(MODELSYMGEN)`

Reply:
No, I messed up here. It ought to be
```
distclean: distclean_assets
```
so that doing plain `distclean` also runs `distclean_assets`. `distclean_kart_assets` is a leftover from the PRs earlier life.

I don't want to put a subcall to `make clean` in `distclean_assets`, its intended to be its own thing, only cleaning out the assets. If someone wants to do a full clean they should run plain `distclean`, while `distclean_assets` is for cleaning out the asset stuff specifically.

Reply:
Why do DLSYMGEN and MODELSYMGEN need to be commented out?

Reply:
Sounds good!

Because the DLSYMGEN and MODELSYMGEN scripts have bugs and we don't need them doing anything unless the game gets modified. Without them running we don't have to deal with broken header files.